components/centos/centos-mesos-marathon-singlenode-setup/provisioning/playbook.yml

```diff
@@ -0,0 +1,38 @@
+- hosts: localhost
+  sudo: yes
+  connection: local
+  tasks:
+          - name: Install Mesosphere repository
+            yum: name=http://repos.mesosphere.com/el/7/noarch/RPMS/mesosphere-el-repo-7-3.noarch.rpm state=present
+
+          - name: Installing zookeeper, mesos, marathon, docker, git
+            yum: name={{item}} state=latest
+            with_items:
+                - mesosphere-zookeeper 
+                - mesos 
+                - marathon
+                - docker
+                - git
+                - epel-release
+                - python-pip
+
+          - name: Starting zookeeper, mesos, marathon, docker
+            service: name={{item}} state=started enabled=yes
+            with_items:
+                - zookeeper
+                - mesos-master
+                - mesos-slave
+                - marathon
+                - docker
+
+          - name: Clone atomicapp
+            git: repo=https://github.com/projectatomic/atomicapp dest=/home/vagrant/atomicapp
+            
+          - name: Add docker containerizer
+            shell: echo 'docker,mesos' > /etc/mesos-slave/containerizers
```

Comment:
In ansible shell command run everytime when provision occur. better use a register variable and add condition.
```
- name: Clone atomicapp
  git: repo=https://github.com/projectatomic/atomicapp dest=/home/vagrant/atomicapp
  register: result

- name: Add docker containerizer
  shell: echo 'docker,mesos' > /etc/mesos-slave/containerizers
  when: result|success
```
Check doc http://docs.ansible.com/ansible/playbooks_conditionals.html